vllm/model_executor/models/olmoe.py

```diff
@@ -212,6 +213,7 @@ def __init__(
             hidden_size=config.hidden_size,
             intermediate_size=config.intermediate_size,
             quant_config=quant_config,
+            prefix=f"{prefix}.moe",
```

Comment:
https://huggingface.co/allenai/OLMoE-1B-7B-0125-Instruct?show_file_info=model.safetensors.index.json
```suggestion
            prefix=f"{prefix}.mlp",
```